recipes/plplot/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "plplot" %}
+{% set version = "5.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://downloads.sourceforge.net/sourceforge/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b92de4d8f626a9b20c84fc94f4f6a9976edd76e33fb1eae44f6804bdcc628c7b
+
+build:
+  number: 0
+  script:
+    - set -ex  # [unix]
+    - CMAKE_ARGS=${CMAKE_ARGS//-DCMAKE_INSTALL_LIBDIR=lib/}  # [unix]
+    - >
+       cmake
+       ${CMAKE_ARGS}  # [unix]
+       -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%  # [win]
+       -DENABLE_octave=OFF
+       -DENABLE_tcl=ON
+       -DENABLE_tk=ON
+       -B_build
+       -GNinja
+    - cmake --build _build
```

Comment:
Is there a `make check` it `ctest` we can run here? Also, while it is tempting to keep everything in a single file, this build lines are complex enough to go to their own `build.sh/bld.bat`

Reply:
Added the examples and tests from plplot. I would prefer keep the script here, since it is a rather generic CMake setup without platform specific details so far.